tmt/cli.py

```diff
@@ -354,9 +354,18 @@ def create(context, name, template, force, **kwargs):
 @click.option(
     '--disabled', default=False, is_flag=True,
     help='Import disabled test cases from Nitrate as well.')
+@click.option(
+    '--manual', default=False, is_flag=True,
+    help='Import manual test cases from Nitrate.')
+@click.option(
+    '--plan', metavar='PLAN',
+    help='Import all manual test cases from Nitrate test plan.')
+@click.option(
+    '--case', metavar='CASE',
+    help='Import manual test case from Nitrate.')
```

Comment:
This should probably say something like "Identifier of test case to be imported."

Reply:
fixed in 440e5c1